packages/block-editor/src/components/rich-text/index.js

```diff
@@ -109,6 +109,7 @@ export function RichTextWrapper(
 		__unstableDisableFormats: disableFormats,
 		disableLineBreaks,
 		__unstableAllowPrefixTransformations,
+		privateDisableEditing,
```

Comment:
I suspect I have taken the private API docs to literally here - they show an example with the new prop prefixed with `private`, but I suspect that negates a bit of the value of the private API, in that once the API is made public we can just unwrap it and all the existing props will just work without having to be renamed?

Reply:
have removed the prefix